crowbar_framework/vendor/assets/stylesheets/jquery/ledUpdate.scss

```diff
@@ -48,3 +48,27 @@
     background: image-url("jquery/ledUpdate/yellow-green-spin.png");
   }
 }
+
+.status-border-unready {
+  border-left: 7px solid #F8E71C!important;
+  animation-name: blink;
+  animation-duration: 2s;
+  animation-iteration-count: infinite;
+}
+@keyframes blink {
+  0% {border-left: 7px green!important;}
+  50% {border-left: 7px solid yellow!important;}
+  100% {border-left: 7px green!important;}
+}
+
+.status-border-ready {
+  border-left: 7px solid #599517!important;
+}
+
+.status-border-failed {
+  border-left: 7px solid #FF001F!important;
+}
+
+.status-border-unknown {
+  border-left: 7px solid #ddd!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used